kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -700,6 +700,14 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(true)
 
+  val FRONTEND_JETTY_SEND_VERSION_ENABLED: ConfigEntry[Boolean] =
+    buildConf("kyuubi.frontend.jetty.send.version.enabled")
+      .doc("When true, Kyuubi will send Jetty version in HTTP response, " +
+        "which may expose vulnerability")
```

Comment:
> which may expose vulnerability

please avoid such thrilling words in user-facing docs

```suggestion
      .doc("Whether to send Jetty version in HTTP response.")
```